client/my-sites/themes/controller.jsx

```diff
@@ -61,25 +61,15 @@ export function upload( context, next ) {
 	next();
 }
 
-export function singleSite( context, next ) {
+export function loggedIn( context, next ) {
 	// Scroll to the top
 	if ( typeof window !== 'undefined' ) {
 		window.scrollTo( 0, 0 );
 	}
 
-	context.primary = <SingleSiteComponent { ...getProps( context ) } />;
-	next();
-}
-
-export function multiSite( context, next ) {
-	const props = getProps( context );
-
-	// Scroll to the top
-	if ( typeof window !== 'undefined' ) {
-		window.scrollTo( 0, 0 );
-	}
+	const Component = context.params.site_id ? SingleSiteComponent : MultiSiteComponent;
```

Comment:
👍 

Shall we rename that route param to `siteId` while we're at it? (Also affects route defs, and probably the `getProps` helper.)